stoplight.js

```diff
@@ -0,0 +1,25 @@
+$(document).ready(function(){
+var redButton=$("")
+var greenButton=$("")
+var yellowButton=$("")
+
+
+function stopFunction(){
+$("#red").toggleclss(.active#red)
+
+}
+function goFunction(){
+
+  $("#green").toggleclass(.active#green)
+}
+function warningFunction(){
+$("#yellow").toggleclass(.active#yellow)
+}
+
+redButton.on("click",stopFunction)
```

Comment:
This is the proper structure of an event listener but the contents of your callback functions will cause this not to work. Take a look at my earlier comments.